### ISSUE TITLE
C0321 false-positive with Python 3.8

### ISSUE THREAD

**Issue by dmtucker** [PyCQA/pylint#2737]
### Steps to reproduce
``` python
"""Module Docstring"""

import click


@click.group(invoke_without_command=True)  # This line seems crucial to the repro.
@click.pass_context
def main(ctx) -> None:
    """Main Docstring"""
    ctx.obj = {}
```
- This is an attempted SSCCE. The real-world presentation can be seen here: https://travis-ci.org/dmtucker/backlog/jobs/491295019#L276

### Current behavior
```
************* Module test
test.py:10:4: C0321: More than one statement on a single line (multiple-statements)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 6.67/10, +0.00)
```
```
pylint 2.2.2
astroid 2.1.0
Python 3.8.0a1+ (heads/master:181835d, Feb 10 2019, 09:29:57) 
[GCC 8.2.0]
```

### Expected behavior
This is the output from Python 3.7:
```

-------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 6.67/10, +3.33)
```
```
pylint 2.2.2
astroid 2.1.0
Python 3.7.2+ (default, Feb  2 2019, 14:31:48) 
[GCC 8.2.0]
```

### pylint --version output
These are included above for clarity, but I can also repro with the latest preview releases:
```
pylint 2.3.0-dev1
astroid 2.2.0-dev
Python 3.8.0a1+ (heads/master:181835d, Feb 10 2019, 09:29:57) 
[GCC 8.2.0]
```

**Comment by PCManticore** [comment]
Thanks for reporting the issue, but we don't support Python 3.8 just yet.

**Comment by PCManticore** [comment]
This has been fixed in the latest `pylint` and `astroid` `master` branches.

**Comment by filips123** [comment]
@PCManticore When will PyLint have full Python 3.8 support on released versions?
